source/MonoGame.Extended.Content.Pipeline/BitmapFonts/BitmapFontWriter.cs

```diff
@@ -41,12 +41,20 @@ protected override void Write(ContentWriter writer, BitmapFontProcessorResult re
 
         public override string GetRuntimeType(TargetPlatform targetPlatform)
         {
+#if KNI
+            return "MonoGame.Extended.BitmapFonts.BitmapFont, KNI.Extended";
```

Comment:
All writers have to return the KNI assembly in GetRuntimeType() and GetRuntimeReader().

Reply:
Is there a way we could refactor this so it's not using `#if` directive and builds it based on the assembly information.  Similar to how MonoGame does here https://github.com/MonoGame/MonoGame/blob/b1244f114a08b103d97c8fcec3e38f27dec93f24/MonoGame.Framework.Content.Pipeline/Serialization/Compiler/BuiltInContentWriter.cs#L46

Reply:
Usually processors are isolated from runtime types, especially for graphics resources, 
e.g. using shadow BitmapFontContent instead of a 'live' BitmapFont.

But here we can use the runtime type information, because we have areference to it. Changed to:
`return typeof(BitmapFont).AssemblyQualifiedName;`